packages/adena-extension/src/repositories/transaction/mapper/transaction-history-mapper.ts

```diff
@@ -176,7 +176,7 @@ export class TransactionHistoryMapper {
       valueType,
       date: dateToLocal(date).value,
       networkFee: {
-        value: `${transfer.amount || '0'}`,
+        value: `${fee.amount || '0'}`,
         denom: `${fee.denom}`,
```

Comment:
we don't need default value for denom?

Reply:
We judge an empty string as an error. 
If a denom is required, it will come in the response.